pkgs/stdenv/generic/setup.sh

```diff
@@ -212,6 +212,14 @@ isELF() {
     if [ "$magic" = $'\177ELF' ]; then return 0; else return 1; fi
 }
 
+isELFExec() {
+  grep -ao -P '^\177ELF.{11}\x00\x02' "$1" >/dev/null
+}
+
+isELFDyn() {
+  grep -ao -P '^\177ELF.{11}\x00\x03' "$1" >/dev/null
```

Comment:
Is the `{11}` going to be the space in bytes?

Reply:
This means we don't care about the 11 bytes following the ELF header (which is 4 bytes).

Reply:
Thanks.